ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -30,39 +35,53 @@
       </v-row>
 
       <div>
-        <p class="text-md-center px-6 pt-3" :class="$style['intro']">
+        <p v-if="!duplicate" class="text-md-center px-6 pt-3" :class="$style['intro']">
           One or more similar {{ partyWord }} Parties were found. Do you want to use an
           existing {{ partyWord }} Party listed below or use your information to create
           a new {{ partyWord }} Party?
         </p>
+        <p v-else-if="!review" class="text-md-center px-6 pt-3" :class="$style['intro']">
+          Registrations cannot list a Secured Party with the same name and address more than once. This registration
+          already contains this Secured Party:
+        </p>
+        <p v-else class="text-md-center px-6 pt-3" :class="$style['intro']">
+          Duplicate Secured Parties have been detected in this registration.<br>
+          Registrations cannot list a Secured Party with the same name and address more than once.<br>
+          <b>Note:</b> these duplicates may not be visible due to a system error.
+        </p>
       </div>
       <div :class="$style['partyWindow']">
-        <div class="text-md-center generic-label" id="create-new-party">
+        <div v-if="!duplicate" class="text-md-center generic-label" id="create-new-party">
           Use my information and create a new {{ partyWord }} Party:
         </div>
 
         <v-container class="currentParty">
-          <v-row :class="[$style['companyRow'], { 'primaryRow': showSelected }]">
-            <v-col cols="auto" :class="$style['iconColumn']"
-              ><v-icon :class="$style['companyIcon']">mdi-domain</v-icon></v-col
-            >
+          <v-row :class="[$style[duplicate ? 'companyRowDuplicate' : 'companyRow'],
+            { 'primaryRow': showSelected && !duplicate }]">
+            <v-col cols="auto" :class="$style['iconColumn']">
+              <v-icon :class="$style['companyIcon']">
+                {{party.businessName ? 'mdi-domain' : 'mdi-account'}}
+              </v-icon>
+            </v-col>
             <v-col cols="9">
               <div :class="$style['companyText']" class="businessName">
-                {{ party.businessName }}
+                {{party.businessName ? party.businessName :
+                party.personName.first+" "+party.personName.middle+" "+party.personName.last}}
               </div>
               <div :class="$style['addressText']">
                 {{ party.address.street }}, {{ party.address.city }}
                 {{ party.address.region }} , {{ party.address.postalCode }},
                 {{ getCountryName(party.address.country) }}
               </div>
               <div>
-                <v-chip x-small label color="primary" text-color="white"
+                <v-chip  x-small v-if="!duplicate" label color="primary" text-color="white"
                   >NEW</v-chip
                 >
```

Comment:
I know not your code, but perhaps keep the tags on the same line. 
The exception is typically a self closing component. ie `<SomeComponent  />` 
But this is a closing tag in itself, so best together. There is a few instances of this as well, try to catch them 👍

Reply:
Quick fixes